### PR TITLE
Rebuild the website daily.

### PR DESCRIPTION
**Pull request by Carreau**
From the documentation this should trigger on the last commit of the
default branch.

This will allow to schedule posts and have them auto published.

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -4,6 +4,10 @@ on:
   push:
     branches:
       - master
+  schedule:
+      # * 6:37 every day – 37 just to not trigger exactly at 6:30 like many other;
+      # 6 is a bit before the beginning of the day in the US.
+      - cron:  '6 37 * * *'
```
Comment (rgommers) [inline .github/workflows/deploy.yml:10]: This looks like it's the wrong way around: https://docs.github.com/en/actions/reference/events-that-trigger-workflows#schedule

https://crontab.guru/#6_37_*_*_*

Comment (Carreau) [reply .github/workflows/deploy.yml:10]:
```suggestion
      - cron:  '37 6  * * *' 
```

Darn, cron should take named arguments.

Comment (rgommers) [reply .github/workflows/deploy.yml]: Comment is also still wrong I think? First link about says the time is in UTC, which means it'll be around midnight in US time zones.

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -4,6 +4,10 @@ on:
   push:
     branches:
       - master
+  schedule:
+      # * 6:37 every day – 37 just to not trigger exactly at 6:30 like many other;
+      # 6 is a bit before the beginning of the day in the US.
+      - cron:  '37 6  * * *'
```
Comment (Carreau) [inline .github/workflows/deploy.yml:10]:
```suggestion
  schedule:
      # * 10:37 every day – 37 just to not trigger exactly at 10:30 like many other;
      #  10:37 UTC is 6:30am EST is a bit before the beginning of the day in the US,
      - cron:  '37 10  * * *' 
```

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (Carreau) [reply .github/workflows/deploy.yml]: Ok, move to 10:37 - 6:37EST – 3:30PST. I think that does not matter much as long as it's at night.
10:30  is still early enough for Europe I think, and we can still tweak.

**Review by rgommers — APPROVED** [review]
Let's give this a try:) Thanks Matthias!

**Comment by rgommers** [comment]
Weirdly this worked for a few days and then failed, without showing any CI run log:

image

No commits were made to master in the meantime.

**Pull request merged**